kalite/distributed/management/commands/setup.py

```diff
@@ -272,6 +272,8 @@ def handle(self, *args, **options):
         # Migrate the database
         call_command("syncdb", interactive=False, verbosity=options.get("verbosity"))
         call_command("migrate", merge=True, verbosity=options.get("verbosity"))
+        # Create *.json and friends database
+        call_command("syncdb", interactive=False, verbosity=options.get("verbosity"), database="topic_tools")
```

Comment:
A 50ms no-op, on my fast desktop computer. So, not too slow, but also unnecessary, no?